members/SCAL8c2b26c2.yaml

```diff
@@ -0,0 +1,65 @@
+bioguide: SCAL8c2b26c2
+contact_form:
+  method: post
+  action: ContactPopupSubmit.php
+  steps:
+    - visit: "https://lcmspubcontact.lc.ca.gov/PublicLCMS/ContactPopup.php?district=AD72&width=600px&height=700px&iframe=true"
+    - fill_in:
+        - name: First Name
+          selector: "#txtFirstName"
+          value: $NAME_FIRST
+          required: true
+          options:
+            blacklist: ".,"
+        - name: Last Name
+          selector: "#txtLastName"
+          value: $NAME_LAST
+          required: true
+          options:
+            blacklist: ".,"
+        - name: Address
+          selector: "#txtAddress"
+          value: $ADDRESS_STREET
+          required: true
+        - name: City
+          selector: "#txtCity"
+          value: $ADDRESS_CITY
+          required: true
+        - name: Phone Area Code
+          selector: "#txtAreaCode"
+          value: $PHONE_AREA_CODE
+          required: false
+        - name: Phone Exchange Code
+          selector: "#txtPhone1"
+          value: $PHONE_EXCHANGE_CODE
+          required: false
+        - name: Phone Subscriber Number
+          selector: "#txtPhone2"
+          value: $PHONE_SUBSCRIBER_NUMBER
+          required: false
+        - name: Zip
+          selector: "#txtZip"
+          value: $ADDRESS_ZIP5
+          required: true
+        - name: Email
+          selector: "#txtEmail"
+          value: $EMAIL
+          required: true
+        - name: Message
+          selector: "#ContactForm textarea[name='jv_text']"
+          value: $MESSAGE
+          options:
+            max_length: 2000
+          required: true
+    - click_on:
+        - value: Submit
+          selector: "#submitButton"
+    - wait:
+        - value: 1
+    - find:
+        - selector: "tr:nth-child(3)"
+  success:
+    headers:
+      status: 200
+    body:
+      contains: "null"
```

Comment:
same comment re success text

Reply:
Oh yeah, forgot to add a comment about that. 

My guess is that forms are still being setup hence why the "null" text after the msg is submitted. 

[![Screenshot from Gyazo](https://gyazo.com/16d9df8b2d4990f90711f31d1f6d7d8f/raw)](https://gyazo.com/16d9df8b2d4990f90711f31d1f6d7d8f)


This is not ideal, as that text will most likely change in the future. I tried to add a step in the yaml to look for "close window" but couldn't make that work. And I also think that adding a `find` for that button could also be an issue later on in case they decide to get rid of that button.

Reply:
This might be a good time to use the `excludes` feature then, since we are fairly sure the success message will not be a reliable thing to look for. Just find text that uniquely appears on the main form but not the thank you page, and do `excludes: "desired text"`

Reply:
good call 
🎉